File: 00-user_defined_data_type/user_defined_data_type.java

public class user_defined_data_type {

	private String name;
	private String id;
	private String adress;
	private float credit;
	
	public static void main(String[] args) {
		user_defined_data_type client_1 = new user_defined_data_type(
											"Anthony Luzquiños",
											"0000000001",
											"Calle 1, Distrito 1, Ciudad 1, País 1", 
											1000000);
		System.out.println(client_1);
	}
	
	public user_defined_data_type(String name, String id, String adress, float credit){
		this.name = name;
		this.id = id;
		this.adress = adress;
		this.credit = credit;
	}
	
	@Override
	public String toString() {
		return "The client's name is: "+this.name+".\n"
				+"The client's id is: "+this.id+".\n"
				+"The client's adress is: "+this.adress+".\n"
				+"The client's credit is: "+this.credit+".";
	}	
}

File: 00-user_defined_data_type/user_defined_data_type.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

struct Client{
    char name[50];
    char id[10];
    char adress[100];
    float credit;
};

int main(int argc, char const argv[]){

    struct Client client_1 = {0};

    strcpy(client_1.name, "Anthony Luzquiños");
    strcpy(client_1.id, "000000001");
    strcpy(client_1.adress, "Calle 1, Distrito 1, Cuidad 1, País 1");
    client_1.credit = 1000000;

    printf("The client's name is: %s.\n", client_1.name);
    printf("The client's id is: %s.\n", client_1.id);
    printf("The client's credit is: %f.\n", client_1.credit);
    printf("The client's adress is: %s.\n", client_1.adress);
    return 0;
}
